### PR TITLE
Feature/line UI area name

### PR DESCRIPTION
**Pull request by fti-dshelke**
Add area name property in lineset data to show it in the center of the line-ui area.

### CODE REVIEW

File: src/LineUI/LineSet.tsx [hunk 1]
```diff
@@ -203,12 +211,27 @@ const LineSet : React.FC<ILineSetProps> = ({ getCTM, boundaries, unit, size, lin
     />
   );});
 
+  const AreaLabel = () => {
+    const pointsLength = lineSetData.points.length;
+    if (pointsLength < 3) return null;
+    let midpoint = { x: 0, y: 0 };
```
Comment (JoshLipps) [inline src/LineUI/LineSet.tsx:217]: const

File: src/LineUI/LineSet.tsx [hunk 2]
```diff
@@ -203,12 +211,27 @@ const LineSet : React.FC<ILineSetProps> = ({ getCTM, boundaries, unit, size, lin
     />
   );});
 
+  const AreaLabel = () => {
```
Comment (JoshLipps) [inline src/LineUI/LineSet.tsx:214]: can we move this component out of the LineSet and just pass props for the points and areaName?

File: src/LineUI/LineSet.tsx (hunk 1 above)
Comment (fti-dshelke) [reply src/LineUI/LineSet.tsx]: We are reassigning value to midpoint

File: src/LineUI/LineSet.tsx (hunk 2 above)
Comment (fti-dshelke) [reply src/LineUI/LineSet.tsx]: Done

**Review by JoshLipps — APPROVED** [review]

File: src/LineUI/LineSet.tsx (hunk 1 above)
Comment (JoshLipps) [reply src/LineUI/LineSet.tsx]: meh ...IMO this means you could use a reduce?

**Pull request merged**